services/bitbucket/bitbucket.tester.js

```diff
@@ -85,6 +92,119 @@ t.create('pr (private repo)')
   .get('/pr/chris48s/example-private-repo.json')
   .expectJSON({ name: 'pull requests', value: 'private repo' })
 
+t.create('pr (server)')
```

Comment:
Can we find any publicly facing projects that would support a live test?

Reply:
I'll look but unless it's a well known company there's no guarantee that it'll stick around.

Reply:
Yeah I haven't found any. Going to mark this as resolved unless someone can find one we can use.